src/layout/Footer.js

```diff
@@ -6,20 +6,35 @@ const Footer = () => {
 	const year = new Date().getFullYear();
 	const iconStyle = "cursor-pointer";
 	return (
-		<div className="bg-primary mt-auto h-24 md:h-32 w-full items-center px-5 text-white grid grid-flow-col">
+		<div className="bg-primary mt-auto p-5 d:h-32 w-full px-5 text-white grid grid-flow-col">
```

Comment:
Maybe you should consider using the `<footer>` tag instead of a `div`.

Reply:
Done!